internal/nginx/config/servers_test.go

```diff
@@ -913,6 +913,42 @@ func TestCreateReturnValForRedirectFilter(t *testing.T) {
 			},
 			msg: "all fields are set",
 		},
+		{
+			filter: &v1beta1.HTTPRequestRedirectFilter{
+				Scheme:     helpers.GetStringPointer("https"),
```

Comment:
better to start the generic version `GetPointer` so that we can get rid of the typed versions